docs/gateway/exchanges/pangolin.md

```diff
@@ -37,3 +37,8 @@ Added by CoinAlpha in [v1.4.0](/release-notes/1.4.0/)
 4. Afterwards, run `create` to create an [AMM Arbitrage](/strategies/amm-arbitrage/) strategy between Pangolin and a different exchange.
 5. Run `start` to start the strategy, and you're trading!
 
+## 📘 Additional Resources
+
+* We recommend using Infura for the RPC node link. Here's a quick guide on how to create the project and get the link - <https://blog.infura.io/post/getting-started-with-infura-28e41844cc89>
```

Comment:
Pangolin is an Avalanche-based DEX, so Infura doesn't work. List the Public API endpoints https://docs.avax.network/apis/avalanchego/public-api-server provided by Avalanche.